node/metrics/src/runtime/mod.rs

```diff
@@ -149,8 +170,8 @@ impl sc_tracing::TraceHandler for RuntimeMetricsProvider {
 				Ok(update_op) => {
 					self.parse_metric_update(update_op);
 				},
-				Err(e) => {
-					gum::error!(target: LOG_TARGET, "TraceEvent decode failed: {:?}", e);
+				Err(_) => {
+					// do nothing
```

Comment:
why remove the error log ?

Reply:
It's misleading, because it never gets logged. Maybe I have to add another comment here explaining this. The one in the top is not visible enough.